src/upgrade.cpp

```diff
@@ -175,6 +175,16 @@ ConfigObject<ConfigValue>* Upgrade::versionUpgrade(const QString& settingsPath)
             delete oldFile;
```

Comment:
Oh, Got It.
So, here is what I did:
- Change local variable name to avoir conflict with oldFile used above
- Wrapped it in QScopedPointer 
- Removed delete
- correct windows #ifdef which was missing "else" statement